example/src/app/a-bit-more-complex-component/a-bit-more-complex.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ABitMoreComplexComponent } from './a-bit-more-complex.component';
+import { ChildComponent } from './child.component';
+
+describe('ABitMoreComplexComponent', () => {
+  let component: ABitMoreComplexComponent;
+  let fixture: ComponentFixture<ABitMoreComplexComponent>;
+
+  beforeEach(
+    async(() => {
+      TestBed.configureTestingModule({
+        declarations: [ABitMoreComplexComponent, ChildComponent],
+      })
+        .overrideComponent(ChildComponent, {set: {template: 'stubbedBody'}})
```

Comment:
By the way. Do you mind adding this approach of overriding components to the docs? This is about shallow rendering.